scalableminds/webknossos
# show all task actions for admin by default

While the issue #1441 only asks to display the `Cancel` action for task for admin, I chose to show all admin actions instead. Hopefully, this makes it less confusing. As admin you can just do anything. 
I double-checked with the backend code and the actions `download`, `cancel`, and `reset` can only be applied if the user's admin right match the task's team. Hence the new check / code.


### Mailable description of changes:
- [Dashboard] Show all actions related to tasks for admins by default

![image](https://user-images.githubusercontent.com/1105056/29719850-21da7f5e-89b8-11e7-91be-b6facea45385.png)

### Steps to test:
1. Go to Dashboard -> Task. Get a task
2. See Trace, Transfer, Cancel, etc
3. Go to Users --> Show Tracings and see all the same actions without "Finish"
4. Sign up new user without admin rights. Only "Trace" and "Finish" should be available, since the user does not have the correct rights.
5. Create a new user with admin rights, but for a different team than the task's team. Only "Trace" and "Finish" should be available.

### Issues:
- fixes #1441
- fixes #1277

------
